### PR TITLE
【code format check upgrade】 step1: pre-commit, remove-ctrlf, pylint

### PR DESCRIPTION
**Pull request by betterpig**
### PR types

Others
### PR changes

Others
### Describe

code format check upgrade step1: pre-commit, remove-ctrlf, pylint
7cc9b336e09064beaaa1fc6ecb441dd4
https://xly.bce.baidu.com/paddlepaddle/paddle/newipipe/detail/5747810/job/15351218

### CODE REVIEW

**Review by luotao1 — APPROVED** [review]

File: paddle/scripts/paddle_build.sh [hunk 1]
```diff
@@ -360,6 +360,10 @@ function check_style() {
     # pre-commit use python3.8.0 
     OLD_PATH=$PATH
     export PATH=/usr/local/python3.8.0/bin:/usr/local/python3.8.0/include:/usr/local/bin:${PATH}
+    
+    if ! [[ $(pre-commit --version) == *"2.17.0"* ]]; then
+        pip install pre-commit==2.17.0
+    fi
```
Comment (luotao1) [inline paddle/scripts/paddle_build.sh:366]: 可以之后把pre-commit, remove-ctrlf, pylint都打入镜像，避免CI 每次重新安装

Comment (luotao1) [inline paddle/scripts/paddle_build.sh:366]: 可以之后把pre-commit, remove-ctrlf, pylint都打入镜像，避免CI 每次重新安装

Comment (betterpig) [reply paddle/scripts/paddle_build.sh:366]: done

Comment (betterpig) [reply paddle/scripts/paddle_build.sh:366]: done

**Pull request merged**